go.mod

```diff
@@ -0,0 +1,132 @@
+	github.com/mholt/archiver v3.1.1+incompatible
+	github.com/munnerz/goautoneg v0.0.0-20190414153302-2ae31c8b6b30 // indirect
+	github.com/nwaples/rardecode v1.0.0 // indirect
+	github.com/onsi/ginkgo v1.6.0
+	github.com/onsi/gomega v1.4.2
+	github.com/pborman/uuid v1.2.0 // indirect
+	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/pierrec/lz4 v0.0.0-20190415111752-9419d2361c8a // indirect
+	github.com/pkg/errors v0.8.1
+	github.com/prometheus/client_golang v0.9.1
+	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90 // indirect
+	github.com/prometheus/common v0.3.0
+	github.com/prometheus/procfs v0.0.0-20190412120340-e22ddced7142 // indirect
+	github.com/robfig/cron v0.0.0-20171101201047-2315d5715e36
+	github.com/satori/go.uuid v1.2.0 // indirect
+	github.com/sirupsen/logrus v1.4.1
+	github.com/smartystreets/goconvey v0.0.0-20190330032615-68dc04aab96a // indirect
+	github.com/soheilhy/cmux v0.1.4 // indirect
+	github.com/spf13/cobra v0.0.3
+	github.com/spf13/pflag v1.0.3
+	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 // indirect
+	github.com/ugorji/go/codec v1.1.7 // indirect
+	github.com/xi2/xz v0.0.0-20171230120015-48954b6210f8 // indirect
+	github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2 // indirect
+	go.etcd.io/bbolt v1.3.3 // indirect
+	golang.org/x/crypto v0.0.0-20190611184440-5c40567a22f8
+	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a // indirect
+	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
+	google.golang.org/appengine v1.5.0 // indirect
+	google.golang.org/genproto v0.0.0-20190418145605-e7d98fc518a7 // indirect
+	google.golang.org/grpc v1.20.1 // indirect
+	gopkg.in/ini.v1 v1.44.0 // indirect
+	gopkg.in/natefinch/lumberjack.v2 v2.0.0-20170531160350-a96e63847dc3 // indirect
+	gopkg.in/yaml.v2 v2.2.2
+	k8s.io/api v0.0.0-20190409021203-6e4e0e4f393b
+	k8s.io/apiextensions-apiserver v0.0.0-20190409022649-727a075fdec8
+	k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
+	k8s.io/apiserver v0.0.0-20190313205120-8b27c41bdbb1
+	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
+	k8s.io/cluster-bootstrap v0.0.0-20190314002537-50662da99b70
+	k8s.io/code-generator v0.0.0-00010101000000-000000000000
+	k8s.io/component-base v0.0.0-20190314000054-4a91899592f4
+	k8s.io/gengo v0.0.0-20190327210449-e17681d19d3a
+	k8s.io/helm v2.7.2+incompatible
+	k8s.io/klog v0.3.1
+	k8s.io/kube-aggregator v0.0.0-20190314000639-da8327669ac5
+	k8s.io/kube-openapi v0.0.0-20190320154901-5e45bb682580
+	k8s.io/utils v0.0.0-20190506122338-8fab8cb257d5
+	sigs.k8s.io/controller-runtime v0.2.0-beta.2
+	sigs.k8s.io/structured-merge-diff v0.0.0-20190302045857-e85c7b244fd2 // indirect
+)
+
+replace (
+	github.com/coreos/etcd => github.com/coreos/etcd v3.3.12+incompatible
+	github.com/gardener/external-dns-management => github.com/gardener/external-dns-management v0.0.0-20190220100540-b4bbb5832a03 // 0.1.0
```

Comment:
Why is this replaced/duplicated here? The same version does also exist in the above list?
Can we get rid of this whole section here?

Reply:
I used replace directives to make sure that the concrete desired version will be picked up. You can note that for some of the them the version in `require` differs from the version in `replace`.